.circleci/config.yml

```diff
@@ -969,6 +1057,67 @@ jobs:
           mentions: <<parameters.slack-mentions>>
           channel: <<parameters.slack-channel>>
 
+  test-e2e-gke:
+    parameters:
+      cluster-id-prefix:
+        type: string
+      mark:
+        type: string
+      parallelism:
+        type: integer
+        default: 1
+      cluster-version:
```

Comment:
Sounds like `gke-version` is less ambiguous?

Reply:
done.